doc/references.bib

```diff
@@ -365,6 +365,16 @@ @article{FischlEtAl2004
   year = {2004}
 }
 
+@article{fishburn2019temporal,
```

Comment:
Please follow the same citekey formatting and capitalisation conventions used for the rest of the entries